apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AudioWorkloadTestRunnerActivity.java

```diff
@@ -60,13 +60,13 @@ public void run() {
             } else {
                 mStatusTextView.setText(getStatus());
                 int result = getResult();
-                String resultText = getResultText();
+                int xRunCount = getXRunCount();
                 if (result == 1) {
-                    mResultTextView.setText("Result: PASS\n" + resultText);
+                    mResultTextView.setText("Result: PASS");
                 } else if (result == -1) {
-                    mResultTextView.setText("Result: FAIL\n" + resultText);
+                    mResultTextView.setText("Result: FAIL\n xRunCount: " + xRunCount);
                 } else {
-                    mResultTextView.setText("Result: UNKNOWN\n" + resultText);
+                    mResultTextView.setText("Result: UNKNOWN\n xRunCount: " + xRunCount);
```

Comment:
Looks like line 62 to line 69 are the same as line 142 to 149. How about making a function for it?